### ISSUE TITLE
Crash with large images

### ISSUE THREAD

**Issue by T8RIN** [usuiat/Zoomable#195]
![image](https://github.com/usuiat/Zoomable/assets/52178347/1813c8d9-1bf3-4d35-ae03-3bdde584fd2a)

**Comment by usuiat** [comment]
Do you use ConstraintLayout?

**Comment by T8RIN** [comment]
Nope

**Comment by T8RIN** [comment]
Maybe 1/4000

**Comment by panpf** [comment]
I recommend a similar image zooming library ZoomImage (https://github.com/panpf/zoomimage).

It perfectly supports Compose Multiplatform and Android View, supports zooming, panning, positioning, rotation, and super-large image subsampling. It also integrates image loaders such as sketch, glide, picasso, coil, etc.

The most important thing is that it can zoom super-large images without memory overflow

**Comment by T8RIN** [comment]
This is good, only for images tho, i will look

**Comment by panpf** [comment]
> This is good, only for images tho, i will look

No, it also works for any Compose component, such as Box, Text, please refer to the documentation: https://github.com/panpf/zoomimage/blob/main/docs/wiki/modifier_zoom.md

**Comment by usuiat** [comment]
I will close this Issue.
If you are still facing problems, please re-open it and attach image files and code that reproduce the problem.
